tests/functional/conftest.py

```diff
@@ -37,3 +37,10 @@ def do_submit_known_email_address(email_address):
         set_email_address(email_address)
         browser.find_by_css('form button').click()
     return do_submit_known_email_address
+
+
+@pytest.fixture
+def click_change_email_link_on_dept_confirm(browser):
```

Comment:
This function name is too long. Ideally, names should contain fewer than 4 words. How about `click_change_email`?